kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -28,6 +28,6 @@ spec:
             memory: 30Mi
             cpu: 100m
           limits:
-            memory: 50Mi
+            memory: 2Gi
```

Comment:
That's pretty drastic. What number of objects do you have in your cluster that you need 2Gi? Generally these manifests will likely always be a suggestion and especially resources must be adapted as needed. Once we benchmark kube-state-metrics properly we can use a vertical auto scaler to make this a bit better, but there will probably never be a one size fits all solution. I'd be fine to bumping there limits/requests to request 150Mi and limit 300Mi, but 2Gi is probably only applicable for larger clusters.